src/mqueryfront/src/SearchJobs.js

```diff
@@ -1,154 +1,129 @@
-import React, { Component } from "react";
+import React from "react";
 import { Link } from "react-router-dom";
-import ErrorBoundary from "./ErrorBoundary";
-import axios from "axios";
-import { API_URL } from "./config";
 import PriorityIcon from "./components/PriorityIcon";
 import ActionClose from "./components/ActionClose";
 import ActionCancel from "./components/ActionCancel";
 import StatusProgress from "./components/StatusProgress";
+import FilteringTableHeader from "./components/FilteringTableHeader";
+import FilteringTitle from "./components/FilteringTitle";
+import PropTypes from "prop-types";
+
+const SearchJobRow = (props) => {
+    const {
+        id,
+        status,
+        submitted,
+        rule_name,
+        priority,
+        files_matched,
+        total_files,
+        files_processed,
+    } = props.job;
+    const rule_author = props.job.rule_author
+        ? props.job.rule_author
+        : "(no author)";
+    const { onClose, onCancel } = props;
+
+    const submittedDate = new Date(submitted * 1000).toISOString();
+
+    const actionBtn =
+        status === "cancelled" || status === "expired" || status === "done" ? (
```

Comment:
Maybe import QueryUtils.js and check for

```suggestion
        finishedStatuses.includes(status)
```

Reply:
I agree. It's the best solution. I'l apply this change in next commit.
My code was based on original one:
"       if (this.props.status === "cancelled" || this.state.cancelled) {
            status = "cancelled";
            cancelBtn = <span />;
        }

        if (this.props.status === "expired") {
            cancelBtn = "";
        }

        if (this.props.status === "done") {
            cancelBtn = "";
        }
"